jaeger-core/src/main/java/io/jaegertracing/internal/SpanContext.java

```diff
@@ -12,10 +12,10 @@
  * the License.
  */
 
-package io.jaegertracing;
+package io.jaegertracing.internal;
```

Comment:
Just wondering what the impact may be on apps that are currently using this class to get the trace/span id? Until the OT spec includes access to this info, it would mean apps using an internal api.

Reply:
I believe the Trace ID/Span ID was never part of the public API. If they were (mis)using it before, they could still misuse it now by making references to this `internal` package.